### PR TITLE
Admin dashboard function

### PR DESCRIPTION
**Pull request by SukhsimranpreetSekhon**

### CODE REVIEW

File: API/admin/reporting/util/countSkillProfiles.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const Sequelize = require("sequelize");
+const Models = require("../../../../models");
+const Skills = Models.skill; // Skills Table
+const Profiles = Models.profile; // Profiles Table
+
+const countSkillProfiles = async () => {
+  const profileSkills = await Skills.findAll({
+    // {Skill ID (id) , descriptionEn, descriptionFr, Count Profile Occurences (countOccurences)}
```
Comment (Rizvida1) [inline API/admin/reporting/util/countSkillProfiles.js:8]: Is this comment necessary ?

File: API/admin/reporting/util/growthRateByMonth.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  const currentDate = moment();
+
+  // console.log("Date: ", currentDate); // Check if initialized
+
+  profiles.forEach(profile => {
+    const profileTime = moment(profile.createdAt);
+    // console.log("Date: ", profileTime);
+    const difference = currentDate.diff(profileTime, "months");
+    // console.log("Difference: ", duration);
+    // console.log(countNewUsers[difference]);
```
Comment (Rizvida1) [inline API/admin/reporting/util/growthRateByMonth.js]: Are these comments necessary ?

File: API/admin/router.js [hunk 3]
```diff
@@ -67,4 +63,7 @@ adminRouter.put(
   admin.updateInactive
 );
 
+// For testing purposes - Reporting BI
```
Comment (Rizvida1) [inline API/admin/router.js]: Not just for testing

**Review by Rizvida1 — DISMISSED** [review]

File: API/admin/reporting/util/growthRateByWeek.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    // console.log(countNewUsers[difference]);
+
+    countNewUsers[difference] = countNewUsers[difference]
+      ? countNewUsers[difference] + 1
+      : 1;
+  });
+
+  countNewUsers = Array.from(countNewUsers, item => item || 0);
+
+  console.log("Array (Weeks): ", countNewUsers);
```
Comment (Rizvida1) [inline API/admin/reporting/util/growthRateByWeek.js]: No logs please

**Review by Rizvida1 — CHANGES_REQUESTED** [review]

File: API/admin/reporting/util/countSkillProfiles.js (hunk 1 above)
Comment (SukhsimranpreetSekhon) [reply API/admin/reporting/util/countSkillProfiles.js:8]: It's just so someone knows how the object is getting returned.

File: API/admin/reporting/util/growthRateByMonth.js (hunk 2 above)
Comment (SukhsimranpreetSekhon) [reply API/admin/reporting/util/growthRateByMonth.js]: Removed.

File: API/admin/router.js (hunk 3 above)
Comment (SukhsimranpreetSekhon) [reply API/admin/router.js]: Removed.

File: API/admin/reporting/util/growthRateByWeek.js (hunk 4 above)
Comment (SukhsimranpreetSekhon) [reply API/admin/reporting/util/growthRateByWeek.js]: Fixed.

**Review by Rizvida1 — APPROVED** [review]

**Pull request merged**